packages/babel-helper-caching/README.md

```diff
@@ -0,0 +1,76 @@
+# @babel/helper-caching
+
+## Usage
+
+The objective of this helper is to create an object that Babel can use as
```

Comment:
Nit: 

"... Babel can use as a unique identifier for the current running logic of a plugin"

or maybe,

"... Babel can use to uniquely identify the current running logic of a plugin"